hazelcast/datadog_checks/hazelcast/data/metrics.yaml

```diff
@@ -548,13 +540,13 @@ jmx_metrics:
         invocationTimeoutMillis:
           alias: hazelcast.member.invocation_timeout_millis
           metric_type: gauge
-        invocations.lastCallId:
+        lastCallId:
```

Comment:
Do these metric attributes only change in 5.x or is this also relevant to 4.x and prior as well. If it doesn't change in 4.x, then we may need to add additional metrics rather than just replacing the existing ones.

Reply:
Hi @Kyle-Neale - is the expectation that every version of Hazelcast is supported? I think Hazelcast 4 is several years old at this point. We can do that, I just want to understand the expectation.

Reply:
@Kyle-Neale I added back support for a local 4.0 test env and made sure this PR doesn't change 4.0 behavior. Some metrics moved and some were removed (apparently). For the ones that were removed I just added another filter to grab them from both the 4.0 and 5.0 locations.

Reply:
I don't think the expectation is to support every version but if it's something that we already supported, I think it makes sense to continue unless there's justification for discontinuing it.